jwst/pipeline/calwebb_spec3.py

```diff
@@ -236,7 +238,7 @@ def process(self, input):
                 for cal_array in result:
                     cal_array.meta.asn.table_name = op.basename(input_models.asn_table_name)
                 result = self.outlier_detection(result)
-
+                result = self.pixel_replace(result)
```

Comment:
Minor nit, but could use a comment line here just saying that pixel_replace is being applied before doing resampling.

Reply:
done